Resources/doc/overriding_controllers.rst

```diff
@@ -0,0 +1,99 @@
+Overriding Default FOSUserBundle Controllers
+============================================
+
+The default controllers packaged with the FOSUserBundle provide a lot of
+functionality that is sufficient for general use cases. But, you might find
+that you need to extend that functionality and add some logic that suits the
+specific needs of your application.
+
+The first step to overriding a controller in the bundle is to create a child
+bundle whose parent is FOSUserBundle. The following code snippet creates a new
+bundle named ``AcmeUserBundle`` that declares itself a child of FOSUserBundle.
+
+.. code-block:: php
+
+    // src/Acme/UserBundle/AcmeUserBundle.php
```

Comment:
this is wrong, as it's before the open tag (which is the wrong line imo, but it's up to FOS to decide that)